.github/workflows/add-remove-labels.yml

```diff
@@ -21,6 +16,12 @@ permissions:
 
 jobs:
   add-remove-labels:
+    if: |
+      contains(github.event.comment.body, '/wip') ||
+      contains(github.event.comment.body, '/verified') ||
+      contains(github.event.comment.body, '/lgtm') ||
+      contains(github.event.comment.body, '/hold') ||
+      contains(github.event.comment.body, '/cherry-pick')
```

Comment:
please add to the welcome message

Reply:
what if i want to cherry-pick only to a specific branch?

Reply:
I will update the welcome message. Your comment should indicate which branch you want to cherry-pick to `/cherry-pick <your branch>`

Reply:
>This ensures, comments added by cherry-pick are not getting 
this is very confusing - marking the comment as packed, is for the user to know the comment is/will be addressed and we should not rely on it for internal functionality

Reply:
Why? If cherry-pick action adds comments like ` ‼️ PR can't be cherry-picked, please merge it first.` or `Cherry pick action created PR $CHERRY_PICK_NUM successfully 🎉!`, why should they get a reaction? Only actions that are going to trigger a workflow should be getting reaction, no?